docs-source/content/userguide/istio/istio.md

```diff
@@ -6,32 +6,31 @@ weight: 1
 
 #### Overview
 
-WebLogic Server Kubernetes Operator version 2.3 and later includes experimental support for Istio 1.2.2 and later.
+WebLogic Server Kubernetes Operator version 2.3 and later includes support for Istio 1.2.2 and later.
 This support allows you to run the operator itself, and WebLogic domains managed by
 the operator, with Istio sidecar injection enabled.  It will allow you to use
 Istio gateways and virtual services to access applications deployed in these domains.
 If your applications have suitable tracing code in them, then you will also be able to
 use distributed tracing, such as Jaeger, to trace requests across domains and to
 other components and services that have tracing enabled.
 
+You can learn more about Istio at [Istio](https://istio.io/latest/docs/concepts/what-is-istio/)
+
 #### Limitations
 
-The current experimental support for Istio has these limitations:
+The current support for Istio has these limitations:
 
-* It is tested with Istio 1.2.2 and later (up to 1.5), however it is tested with both single and
+* It is tested with Istio 1.2.2 and later (up to 1.5.4), however it is tested with both single and
   multicluster installations of Istio.
-* Support is provided only for domains that are stored in persistent
-  volumes and created with the provided sample using the WLST option.
-  We intend to support Domain in Image and WDT options as well, but that is not currently
-  available.
-* Support is provided only for domains with a single dynamic cluster.
-  Multiple clusters and configured clusters are not currently supported.
+* You cannot expose the default channel, any attempt will result in an error when deploying the domain.  
```

Comment:
the default channel -> any of the default channels

Reply:
fixed